src/assets.ts

```diff
@@ -685,26 +696,29 @@ async function deleteAssets(generator: AssetGenerator) {
     ]);
 }
 
-async function shouldGenerateAssets(generator: AssetGenerator): Promise<boolean> {
-    return new Promise<boolean>((resolve, reject) => {
+async function shouldOverwriteAssets(generator: AssetGenerator): Promise<'Yes' | 'No' | 'Cancel'> {
+    return new Promise<'Yes' | 'No' | 'Cancel'>((resolve, reject) => {
         doesAnyAssetExist(generator).then(res => {
             if (res) {
                 const yesItem = { title: 'Yes' };
+                const noItem = { title: 'No' };
                 const cancelItem = { title: 'Cancel', isCloseAffordance: true };
-                vscode.window.showWarningMessage('Replace existing build and debug assets?', cancelItem, yesItem)
+                vscode.window.showWarningMessage('Replace existing build and debug assets?', cancelItem, noItem, yesItem)
                     .then(selection => {
                         if (selection === yesItem) {
-                            resolve(true);
+                            resolve('Yes');
+                        }
+                        else if (selection === noItem) { 
+                            resolve('No');
                         }
```

Comment:
What is your design here? Since you aren't deleting the files anymore (and it doesn't seem like we should), I am not following the goal. Potentially I could imagine this have the behavior of only adding missing sections, but it doesn't seem to be doing this and I am not sure that actually makes sense.

Reply:
In my comment above: 

> The prompt I changed in my first commit to be Cancel / No / Yes, turned out to be very confusing. Because we are now trying hard to not overwrite anything, can I just get rid of the prompt, and if there is existing task just rename it to "* - backup"?

This prompt is a leftover from the first concept, right now I was just trying to figure out how to go about this task. If we should append the "- Backup" or prompt to overwrite. In the end think this prompt needs to revert to what it was. 

This would be my current goal: 

Check if there are any configurations or tasks that we would overwrite and get their names. If there are any, prompt with Cancel | Yes.

On Cancel do nothing. 
On Yes, patch the files.

If there are none, patch the files, no prompt.

How exactly the files are patched is a different thing to figure out.

Reply:
Your suggestion of a backup file certainly sounds reasonable. The other option that occurs to me is to change the prompt to be more clear on what we are overwriting -- "Replace existing build tasks and .NET debug launch configurations?"

Reply:
This is outdated, I ended up doing pretty much the same thing as before. It checks if we overwrite any task, and prompts for confirmation. I experimented with printing the names of tasks in the prompt but it was confusing and noisy.